### ISSUE TITLE
1.x-1.x.x official release?

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/radix#7]
@arshad are you interested in making an official release so that Radix shows up on the official backdropcms.org website?

**Comment by shadcn** [comment]
Yep of course. I'll work on this today. Do we have a guide / documentation?

**Comment by jenlampton** [comment]
That would be a good idea..  I'll make a guide here https://api.backdropcms.org/creating-official-releases
